api/envoy/config/core/v3/protocol.proto

```diff
@@ -157,6 +157,10 @@ message HttpProtocolOptions {
   // If this setting is not specified, the value defaults to ALLOW.
   // Note: upstream responses are not affected by this setting.
   HeadersWithUnderscoresAction headers_with_underscores_action = 5;
+
+  // Optional maximum requests for both upstream and downstream connections.
+  // If not specified, there is no limit. It's a soft limit.
```

Comment:
What does soft limit mean here?

Reply:
Not failing a request, if the limit exceeds.

Reply:
I would describe in more detail the fact that HTTP/2 and HTTP/3, due to concurrent stream processing, the limit is approximate.